docs/docs/types/gate.md

```diff
@@ -0,0 +1,46 @@
+A more invasive approach to data collection or user identification. This type of module prevents users from viewing content or navigating until some sort of data has been collected and processed. Highly effective when promoting white papers or other online materials that are not to be freely available.
```

Comment:
Invasive is accurate, but a bit negative. Maybe "An approach for requiring data collection or user identification".